spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRemoveOrphanFilesAction3.java

```diff
@@ -61,6 +61,8 @@ public void testSparkCatalogTable() throws Exception {
     assertThat(StreamSupport.stream(results.orphanFileLocations().spliterator(), false))
         .as("trash file should be removed")
         .anyMatch(file -> file.contains("file:" + location + trashFile));
+
+    cat.purgeTable(id);
```

Comment:
the initial approach to avoid this issue in the first place was to have a randomized part in the table name via `ThreadLocalRandom.current().nextInt(1000)`. Maybe we should update this to have a random UUID instead?

My concern with calling `cat.purgeTable(id)` is that this code isn't necessarily reached when the test fails in between for other reasons

Reply:
there are a few places in this class and in its super class where this is being done, so might be worth adding a small method that does `UUID.randomUUID().toString().replace("-", "")` and appends this to the passed table name